src/compiler/diagnosticMessages.json

```diff
@@ -1755,6 +1751,10 @@
         "category": "Error",
         "code": 2535
     },
+    "The 'arguments' object cannot be referenced in a generator function or method in ES3 and ES5. Consider using a standard function or method.": {

```

Comment:
Instead of suggesting using a standard function, why not suggest using a rest parameter, ie `...args: any[]`?